Recast/Source/Recast.cpp

```diff
@@ -307,7 +359,7 @@ void rcClearUnwalkableTriangles(rcContext* ctx, const float walkableSlopeAngle,
 		calcTriNormal(&verts[tri[0]*3], &verts[tri[1]*3], &verts[tri[2]*3], norm);
 		// Check if the face is walkable.
 		if (norm[1] <= walkableThr)
-			areas[i] = RC_NULL_AREA;
+			areaMod.apply(areas[i]);
```

Comment:
As I mentioned previously - `RC_NULL_AREA` has a special meaning to the meshing process, so I do not think we should take `areaMod` here.